app/controllers/assignments_controller.rb

```diff
@@ -60,7 +60,9 @@ def render_test_result
   def student_interface
     @assignment = Assignment.find(params[:id])
     if @assignment.is_hidden
-      render file: "public/404.html", status: 404
+      render 'shared/http_status', formats: [:html],
+             locals: { code: '404', message: HttpStatusHelper::ERROR_CODE['message']['404'] },
```

Comment:
Line is too long. [94/80]<br>Align the elements of a hash literal if they span more than one line.